README.md

```diff
@@ -31,22 +31,46 @@ EOF
 oc create -f ssh-secret.yml
 ```
 
+The CI pipeline supports a custom private registries where bundle and test
+index are pushed. To allow a private registry access user need to create
+auth secret - .dockerconfigjson with registry username and access token.
+
+```bash
+# Download an access token and create a secret
+
+cat << EOF > redhat-isv-redhat-isv-robot-secret.yml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: redhat-isv-redhat-isv-robot-pull-secret
+data:
+  .dockerconfigjson: < ACCESS CREDENTIALS >
+type: kubernetes.io/dockerconfigjson
+EOF
+
+oc create -f redhat-isv-redhat-isv-robot-secret.yml
+```
+
 To trigger a CI pipeline follow steps below:
 ```bash
 oc apply -R -f pipelines/operator-ci-pipeline.yml
 oc apply -R -f tasks
 
-# Install external yaml-lint task
+# Install external dependencies
 curl https://raw.githubusercontent.com/tektoncd/catalog/main/task/yaml-lint/0.1/yaml-lint.yaml | oc apply -f -
+curl https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.4/git-clone.yaml | oc apply -f -
 
 
 tkn pipeline start operator-ci-pipeline \
-  --param git_repo_url=https://github.com/Allda/operator-test-repo.git \
-  --param git_repo_name=Allda/operator-test-repo \
-  --param git_revision=master \
+  --param git_repo_url=git@github.com:redhat-openshift-ecosystem/operator-pipelines-test-repo.git \
```

Comment:
I think you should do that for both of the pipelines

Reply:
Added

Reply:
Thank you!